api-tester/src/main/java/com/revenuecat/apitester/java/StoreProductAPI.java

```diff
@@ -6,24 +6,29 @@
 import com.revenuecat.purchases.amazon.AmazonStoreProductKt;
 import com.revenuecat.purchases.models.GoogleStoreProduct;
 import com.revenuecat.purchases.models.GoogleStoreProductKt;
+import com.revenuecat.purchases.models.Period;
 import com.revenuecat.purchases.models.Price;
-import com.revenuecat.purchases.models.SubscriptionOption;
 import com.revenuecat.purchases.models.StoreProduct;
-import com.revenuecat.purchases.models.Period;
+import com.revenuecat.purchases.models.SubscriptionOption;
 import com.revenuecat.purchases.models.SubscriptionOptions;
 
 import org.json.JSONObject;
 
+import java.util.Locale;
+
 @SuppressWarnings({"unused"})
 final class StoreProductAPI {
     static void check(final StoreProduct product) {
+        final Locale locale = Locale.getDefault();
         final String productId = product.getId();
         final String sku = product.getSku();
         final ProductType type = product.getType();
         final Price price = product.getPrice();
         final String title = product.getTitle();
         final String description = product.getDescription();
         final Period period = product.getPeriod();
+        final String pricePerMonth = product.formattedPricePerMonth(locale);
```

Comment:
Note that I haven't added a version of this without the `locale` parameter. The reason for this is that `StoreProduct` is an interface, and we can't use `@JvmOverloads` there. So we would need to create 2 functions, one with the parameter and one without to support this in Java.

I thought that it wasn't necessary since getting the locale should be easy enough, but lmk if you think we should do that

Reply:
The fact that `pricingPhase.formattedPricePerMonth` has the overloads makes it look a bit inconsistent, but it's only for Java users, so I am not super worried.